.eslintrc.json

```diff
@@ -1,21 +1,19 @@
 {
-    "env": {
-        "browser": true,
-        "commonjs": true,
-        "es6": true
-    },
-    "extends": [
-        "google"
-    ],
-    "globals": {
-        "Atomics": "readonly",
-        "SharedArrayBuffer": "readonly"
-    },
-    "parserOptions": {
-        "ecmaVersion": 2018
-    },
-    "rules": {
-        "max-len": ["error", { "code": 120 }],
-        "comma-dangle": "off"
-    }
-}
\ No newline at end of file
+  "env": {
+    "browser": true,
+    "commonjs": true,
+    "es6": true
+  },
+  "extends": ["prettier"],
+  "globals": {
+    "Atomics": "readonly",
+    "SharedArrayBuffer": "readonly"
+  },
+  "parserOptions": {
+    "ecmaVersion": 2018
+  },
+  "rules": {
+    "max-len": ["error", { "code": 120 }],
+    "comma-dangle": "off"
```

Comment:
Might no longer need this rule since prettier will remove all trailing commas anyway!